src/piaf/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Article(models.Model):
+    CHOICES = [('pending', 'pending'), ('progress', 'progress'), ('completed', 'completed')]
+    name = models.CharField(max_length=100)
+    status = models.CharField(max_length=10, choices=CHOICES, default='pending')
+    user = models.ForeignKey(User, null=True, on_delete='null')
+    participated_at = models.DateField(auto_now=True)
+
+
+class Paragraph(models.Model):
+    article = models.ForeignKey(Article, on_delete='cascade', related_name='paragraphs')
+    text = models.TextField()
+
+
+class Question(models.Model):
+    paragraph = models.ForeignKey(Paragraph, on_delete='cascade', related_name='questions')
+    text = models.CharField(max_length=200)
```

Comment:
We need the User ID of the question don't we ?
And a "created_at" field to track the timings ?

Reply:
Right now a question is linked to a paragraph, that is linked to an article, that is assigned to a user. Therefore questions are indirectly linked to users.
Only one user can read and ask questions.
However, if an article could be split through multiple users (one paragraph per user) as we discussed earlier, then a paragraph should be linked and not an article.

Considering this, I may be missing something here. Do you see some reason to assign a User to a Question?

Reply:
![Screen Shot 2019-10-07 at 14 20 24](https://user-images.githubusercontent.com/13087916/66310952-ab579080-e90d-11e9-8d12-73209680bfcf.png)

Here is a little schema with some numbers about the models. So you are right. Only one user per question, and it should come from the paragraph, coming from the article. 
The answers,

However, I try to anticipate the QC we will do... And maybe it would be easier on the phone for this.